src/main/java/edu/oregonstate/cs361/battleship/Ship.java

```diff
@@ -35,17 +34,17 @@
         type = "Ship";
     }
 
-    String getName() { return name; }
+    public final String getName() { return name; }
 
-    int getLength() { return length; }
+    public final int getLength() { return length; }
 
-    Coords getStart() { return start; }
+    protected final Coords getStart() { return start; }
```

Comment:
Why are some of these protected rather than public? Probably doesn't make a difference because they're all in the same package.